api.hpp

```diff
@@ -739,6 +740,13 @@ class Client {
   Socket sock_ = (Socket)-1;
   std::vector<NettestFlags> granted_suite_;
   Settings settings_;
+
+  nlohmann::json web100_;
+  double download_speed_;
+  double upload_speed_;
+  double download_retrans_;
+  double upload_retrans_;
+
```

Comment:
You're going to want to add MinRTT to this. Do you want to do it in this PR, or later?

Either way, I recommend you make a struct called "summary_data" and add these 4 elements to it.   Then it is clear that these data elements are all related (which they are), and what their purpose is.

Reply:
You're right, and I preferred adding MinRTT as part of this PR. PTAL again?

Reply:
Some of your output is JSON and some is not.  Please either make it all JSON or make none of it JSON. Half and half is not helpful.  Possibly the right thing is for you to pretty-print the JSON so it is human-readable?